libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -173,6 +189,44 @@ bool AP_ESC_Telem::get_current(uint8_t esc_index, float& amps) const
     return true;
 }
 
+// get the highest ESC current if available, returns true if there is valid data for at least one ESC
+bool AP_ESC_Telem::get_highest_current(float& amps) const
```

Comment:
Why do you want highest current rather than total current across all ESCs?

Reply:
To help stay under the ESCs current rating but indeed I wanted to also have the possibility to display total current I forgot to add it

Reply:
I added total current too